main.go

```diff
@@ -83,6 +84,9 @@ func main() {
 	// init log format from envvar ESTAFETTE_LOG_FORMAT
 	foundation.InitLoggingFromEnv(foundation.NewApplicationInfo(appgroup, app, version, branch, revision, buildDate))
 
+	// create context to cancel commands on sigterm
+	ctx := foundation.InitCancellationContext(context.Background())
```

Comment:
@ivanovaleksandar here you don't need to add initCancellationContext as we are passing foundation.InitGracefulShutdownHandling() in the line 118. just you can create the context with 
ctx: = context.Background()

Reply:
@msharbaji that was my understanding as well;)

Reply:
We discussed this with @snigdhasambitak little bit already above, but I can create it as just a normal context. Doesn't have to be one with cancellation.

Reply:
ok, created normal context instead now.